cpp/core/tests/ShuffleSplitTest.cc

```diff
@@ -51,7 +51,7 @@ class MyMemoryPool final : public arrow::MemoryPool {
 
   Status Allocate(int64_t size, int64_t alignment, uint8_t** out) override {
     if (bytes_allocated() + size > capacity_) {
-      return Status::OutOfMemory("malloc of size ", size, " failed");
+      return arrow::Status::OutOfMemory("malloc of size ", size, " failed");
```

Comment:
Maybe you can use `using namespace arrow` in this .cc file

Reply:
:)